packages/cras.rb

```diff
@@ -55,16 +50,21 @@ def self.build
 }
 _EOF_'
       system './git_prepare.sh'
+      ENV['CC'] = 'clang'
+      ENV['CXX'] = 'clang++'
+      ENV['CXXFLAGS'] = "-g -O2 -std=gnu++11 -Wall"
+      ENV['CFLAGS'] = "-g -O2 -Wall"
```

Comment:
Do we really need clang?  Wouldn't that require `depends_on 'llvm' => :build`.

Reply:
It failed while compiling with gcc. Don't know why... Indeed llvm is installed on my machine, so did not thought about that.